client/src/components/Common/MessageChannel.tsx

```diff
@@ -1,11 +1,8 @@
 import React from 'react'
 import Message from './ChatMessage'
 import { FlatList, StyleSheet, View } from 'react-native'
-import { MessageType } from '../../types/Message'
```

Comment:
MessageType already exists within the Message.ts file.

Maybe migrate it over to your file.

Also delete your Message type